duckduckgo/Android
# Move download improvements from Internal to everyone

Task/Issue URL: https://app.asana.com/0/0/1201717030838678/f

### Description
Made the new approach for downloads available to everyone.

### Steps to test this PR

- fresh install from this branch using the Play flavor
- download this image: https://miro.medium.com/max/1050/0*BRkMUINdhCYPk1KM.jpg
- notice it works as expected
- download this pdf: https://www.jdwetherspoon.com/~/media/files/pdf-documents/menus/currentmenus/menu_2103.pdf
- notice it works as expected


### NO UI changes
